.github/workflows/codeql.yaml

```diff
@@ -14,6 +15,8 @@ jobs:
     with:
       image-base-name: "dev_image"
   analyze-go:
+    # go analysis is a bit slow and so skip it for PRs but run it on main
+    if: ${{ !github.event.issue.pull_request }}
```

Comment:
I think its just `github.event.pull_request`. `github.event.issue.pull_request` is for issue_comments on pull requests.

Reply:
https://docs.github.com/en/webhooks-and-events/webhooks/webhook-events-and-payloads#pull_request

Reply:
Yeah noticed that it didn't work. Switched over to `event_name` instead.